### PR TITLE
All Spec Harness Tests Passing

### PR DESCRIPTION
**Pull request by LHJE**
Great work team

### CODE REVIEW

**Comment by PhilipDeFraties** [comment]
Nice! Would this be a good time to merge to main?

**Review by PhilipDeFraties — APPROVED** [review]
Awesome! Pretty crazy that the problems were also very similar to each other for each method when writing to pass specharness.

File: lib/season_stats.rb [hunk 1]
```diff
@@ -27,26 +27,23 @@ def most_accurate_team(the_season)
     this_season = HelperMethods.find_this_season(the_season)
     this_season_game_ids = HelperMethods.find_games_by_game_id(this_season)
     game_list = HelperMethods.find_game_list(game_teams_by_id, this_season_game_ids)
-    flattened_game_list = game_list.flatten
-    teams_by_id = HelperMethods.find_teams_by_team_id(flattened_game_list)
+    teams_by_id = HelperMethods.find_teams_by_team_id(game_list)
     team_and_accuracy = HelperMethods.find_team_and_accuracy(teams_by_id)
     best_team = HelperMethods.largest_hash_value(team_and_accuracy)
-    team_name = HelperMethods.find_team_name_for_accuracy(best_team)
+    team_name = HelperMethods.find_team_name(best_team)
```
Comment (PhilipDeFraties) [inline lib/season_stats.rb:33]: I can't believe I thought my find_team_name version was working lol.

Comment (LHJE) [reply lib/season_stats.rb:33]: Mine still needed tweaking too so same hereee!

File: lib/helper_methods.rb [hunk 2]
```diff
@@ -141,23 +142,26 @@ def self.find_result_games_this_season(this_season, result_games_with_key_as_gam
...
+        result_games_by_game_id[game_this_season.game_id] = game_this_season.game_id
       end
     end
-    # binding.pry
+
     result_games_by_game_id.values
   end
 
-  def self.find_game_list(games_with_key_as_game_id, games_by_game_id)
+
```
Comment (BJSherman80) [inline lib/helper_methods.rb:155]: remove white space

File: lib/league_stats.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 require_relative './helper_methods'
-
+require 'pry'
 class LeagueStats
```
Comment (BJSherman80) [inline lib/league_stats.rb:4]: remove pry

**Review by BJSherman80 — APPROVED** [review]

**Pull request merged**